doc/configurations/experiments.rst

```diff
@@ -69,6 +69,28 @@ For an overview of common compsets used for NorESM2 CMIP6 experiments, please se
 
 For an overview of the compsets provided for CESM2, please see: http://www.cesm.ucar.edu/models/cesm2/config/compsets.html.
 
+Emission driven compsets
+''''''''''''''''''''''''
+NorESM2 can be run in emission driven mode for interactive carbon-cycle studies. Currently, this configuration is only supported for the LM-resolution. In order to run NorESM2-LM in emission driven mode, the biogeochemical physics ``[_BGC]%pys`` is set to  ``%BPRPDMS`` (instead of ``%BDRDDMS``)
+
+There exists some predefined emission driven compsets which usually ends with **esm** :
+
+* **N1850esm**: Emission driven coupled configuration for NorESM for pre-industrial (1850) conditions
+* **NHISTesm**: Emission driven historical configuration from 1850 up to year 2015 
+
+In order to run other experiments in emission driven mode, you can simply replace `%BDRDDMS` with `%BPRPDMS` in ``<noresm_base>/cime_config/config_compsets.xml``, e.g. to create an emission driven SSP5-8.5 scenario compset: ::
+
+ <compset>
+    <alias>NSSP585frc2esm</alias>
+    <lname>SSP585ESM_CAM60%NORESM%FRC2_CLM50%BGC-CROP_CICE%NORESM-CMIP6_MICOM%ECO_MOSART_SGLC_SWAV_BGC%BPRPDMS</lname>
+  </compset>
+
+
+which you then can use when you create your new case:::
+
+./create_newcase --case path_to_cases/NSSP585frc2esm_f19_tn14 --compset NSSP585frc2esm --res f19_tn14 --mach betzy --project project_nr --run-unsupported
+
```

Comment:
Perhaps you can also add:

With the exception of preindustrial control, running in CO2 emission-driven mode requires time-varying spatial CO2 emissions boundary condition files. Currently, only historical and ssp585 scenarios have been tested.

Reply:
Thanks @JorgSchwinger ! There have been some questions about how to run NorESM in emission-driven mode, so I wanted to include a short description in the documentation. However, I have never run NorESM in emission-driven mode so I'm no expert on this. But I think it is useful to add some description. Maybe we can remove the part with *making your own compsets* for now and include that part later if you or @tjiputra are willing to make a description?  We can still add the two sentences Jerry suggested. Have to tested ssp585 in 2.0.5 @tjiputra ?